web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -107,6 +107,9 @@
     -->
     <xsl:variable name="isRequired" as="xs:boolean">
```

Comment:
Check also in https://github.com/geonetwork/core-geonetwork/blob/3.4.x/schemas/iso19139/src/main/plugin/iso19139/layout/layout-custom-fields.xsl#L82, this template in the iso19139 schema has similar logic to calculate there the `isRequired` value, will require update also.